lib/Github/Exception/ApiLimitExceedException.php

```diff
@@ -9,8 +9,24 @@
  */
 class ApiLimitExceedException extends RuntimeException
 {
-    public function __construct($limit = 5000, $code = 0, $previous = null)
+    private $limit;
+    private $reset;
+
+    public function __construct($limit = 5000, $reset = 1800, $code = 0, $previous = null)
```

Comment:
I would not add the new argument in the middle. It requires everyone using the exception themselves to update their code.